larger_than_memory.ipynb

```diff
@@ -0,0 +1,714 @@
+{
```

Comment:
<p>Considering to write file to disk, and then read it from there, to have a "reading a file" case here</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/ehrapy-tutorials/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Not sure whether we need it but it doesn't hurt</p>